docs/src/docs/next-steps.md

```diff
@@ -1,11 +1,19 @@
 # Next Steps with Kit
 
 In this guide you'll learn how to:
+* Sign your ModelKit
 * Make your own Kitfile
 * The power of `unpack`
 * Read the Kitfile or manifest from a ModelKit
 * Tag ModelKits and keep your registry tidy
 
+## Signing your ModelKit
+
+Because ModelKits are OCI artifacts, they can be signed the same way you sign your other containers.
+
+If you need a quick way to sign your containers, check out [Cosign](https://docs.sigstore.dev/signing/quickstart).
```

Comment:
Same here - say something more like "...way to sign a ModelKit you can follow the same instructions as for a container using something like [Cosign]"

Reply:
I made the change you suggested here, too.